packages/pluggableWidgets/popup-menu-web/src/PopupMenu.editorConfig.ts

```diff
@@ -0,0 +1,19 @@
+import { hidePropertyIn, Properties } from "@widgets-resources/piw-utils";
+import { BasicItemsPreviewType, PopupMenuPreviewProps } from "../typings/PopupMenuProps";
+
+export function getProperties(values: PopupMenuPreviewProps, defaultProperties: Properties): Properties {
+    if (values.renderMode === "basic") {
+        hidePropertyIn(defaultProperties, values, "customItems");
+
+        values.basicItems.forEach((item: BasicItemsPreviewType, index: number) => {
+            if (item.itemType === "divider") {
+                hidePropertyIn(defaultProperties, values, "basicItems", index, "caption");
```

Comment:
you can use **hidepropertiesin** which will accept array of keys

Reply:
But that function doesn't allow `nestedPropKey` right? So I'm not sure if I can